### ISSUE TITLE
Feature request to support FreshRSS 

### ISSUE THREAD

**Issue by boncuk10** [prof18/feed-flow#41]
Hi, the app is really great but I cannot login to my FreshRSS account. Could you check if it’s possible to support FreshRSS?

**Comment by prof18** [comment]
Hi, thanks for the suggesting! I'm already investigating the possibility of adding external services.

Just a question: would you be expecting complete support, for example with the read/unread items, or just for having a source of truth for the feed sources?

**Comment by boncuk10** [comment]
Have you had a chance to see if you can support FreshRSS with read and unread items? I use following instance: https://frss.adminforge.de/

**Comment by prof18** [comment]
Not yet, but it's on the timeline!

**Comment by AndreaDiNapoli** [comment]
This would be awesome!

**Comment by twhite96** [comment]
YES. Any progress on this? I know a tiny bit of Java (not sure if Android still uses Java proper or just Kotlin now) but maybe I can take a look

**Comment by prof18** [comment]
I've started documenting myself and doing some architectural planning, so it's going. But I cannot provide any ETA, it will take some time 😅

**Comment by 42Craft** [comment]
yes please!

**Comment by boncuk10** [comment]
Hi Marco, just wanted to ask if you could share with us any status regarding FreshRSS support? Thanks in advance 👍

**Comment by 42Craft** [comment]
> Hi Marco, just wanted to ask if you could share with us any status regarding FreshRSS support? Thanks in advance 👍

I'm using Feed Me with FreshRss

**Comment by prof18** [comment]
> Hi Marco, just wanted to ask if you could share with us any status regarding FreshRSS support? Thanks in advance 👍

it's in progress! No precise ETA, but could be one of the next features, so stay tuned 😁

**Comment by rmdes** [comment]
I'm testing it now, I know it's in beta, so I'm happy to provide feedback

I'm mostly interested in being able to sync my existing collection, then having support for read/unread would be nice and ability to star etc..

But for now, I can't login to my own Instance.
No matter the URL I set, with or without the api and using greader.php or not..

Am I missing something?

**Comment by rmdes** [comment]
After login in, I can see my categories on the left pane, but feeds never load... After a while I see some kind of a pop-up saying

`sorry something went wrong while syncing data in your account :(`

**Comment by prof18** [comment]
Thank you! I'll check the crash and I'll try to debug it. Otherwise, I'll make another update with proper error codes in the messages to try to pinpoint the issue. 

Thank you for reporting it!

**Comment by prof18** [comment]
Also I'll maybe add what you said in the app, just to help clarify

https://github.com/prof18/feed-flow/issues/41#issuecomment-2634686433

**Comment by prof18** [comment]
@rmdes Would you mind dumping your feeds into an OPML and sharing it via email? So, I'll try to set an instance with that.

**Comment by prof18** [comment]
@rmdes The latest version should have fixed all the issues

**Comment by rmdes** [comment]
> @rmdes The latest version should have fixed all the issues

I confirm 💙

It's working smoothly now, it's really neat, thanks a lot for your hard work 🙏

**Comment by JacobDB** [comment]
Wow, what a coincidence, I just set up a FreshRSS install this week and I see this was only completed in the last month! Love the app, just got the connection set up, looks great!

For anyone stumbling across this, you can find the documentation for the FreshRSS API here: 

https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html

**Comment by shuvashish76** [comment]
@Alkarex Request to add this project to FreshRSS [APIs & native apps list](https://github.com/FreshRSS/FreshRSS#apis--native-apps).

Edit: reference https://github.com/FreshRSS/FreshRSS/issues/7301
